juntagrico/admins/forms/subscription_admin_form.py

```diff
@@ -14,8 +13,9 @@ class Meta:
         model = Subscription
         fields = '__all__'
 
-    subscription_members = forms.ModelMultipleChoiceField(queryset=MemberDao.all_members(), required=False,
-                                                          widget=admin.widgets.FilteredSelectMultiple('Member', False))
+    subscription_members = forms.ModelMultipleChoiceField(
+        queryset=MemberDao.all_members(), required=False, widget=admin.widgets.FilteredSelectMultiple('Member', False),
+        label=Subscription.recipients_names.short_description)
```

Comment:
macht es nicht mehr sinn den string gleich hier reinzuschreiben als den umweg über die klasse zu machen?

Reply:
Das eine passt den namen in der Listenansicht und der andere in der Bearbeitungsansicht an. Und sie sollten beide gleich sein.

Reply:
ok